### PR TITLE
[12.0][MIG] account_fiscal_year

### PR DESCRIPTION
**Pull request by rven**
Depends on [12.0][MIG] date_range #25

### CODE REVIEW

File: account_fiscal_year/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Account Fiscal Year',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'author': 'Camptocamp SA,'
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.camptocamp.com',
```
Comment (nikul-serpentcs) [inline account_fiscal_year/__manifest__.py]: @rven Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

File: account_fiscal_year/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Author: Damien Crier
```
Comment (aitorbouzas) [inline account_fiscal_year/__manifest__.py:1]: Maybe this should go to 'author' section.

File: account_fiscal_year/models/date_range_type.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Author: Damien Crier
```
Comment (aitorbouzas) [inline account_fiscal_year/models/date_range_type.py:1]: Same as above.

File: account_fiscal_year/models/res_company.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Author: Damien Crier
```
Comment (aitorbouzas) [inline account_fiscal_year/models/res_company.py:1]: Same as above.

**Review by aitorbouzas — COMMENTED** [review]
LGTM, just some small changes.
Damien Crier is from Camptocamp so it shouldn't apear as ```# Author: Damien Crier```
Check this: [URL](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L2)

**Comment by aitorbouzas** [comment]
Also, ```date_range``` is already available in V12 in @OCA ```server-ux```

**Comment by pedrobaeza** [comment]
I think this module is no longer needed, as now Odoo allows to put variable fiscal years. Or this feature is enterprise only? @fclementic2c can you confirm?

**Comment by aitorbouzas** [comment]
@pedrobaeza You are right!. If you mean [this](https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_fiscal_year.py#L11) it's CE.

**Comment by pedrobaeza** [comment]
Yes, that's it

**Comment by eLBati** [comment]
@aitorbouzas @pedrobaeza so maybe we should create a menuitem pointing to `account.actions_account_fiscal_year`, as the action is present but hidden (shown by an enterprise menu)

**Comment by pedrobaeza** [comment]
@eLBati yes this module can do that

**Comment by eLBati** [comment]
@aitorbouzas @pedrobaeza adding refactored `account_fiscal_year` with https://github.com/OCA/account-financial-tools/pull/726

**Comment by pedrobaeza** [comment]
Closing this one then.

**Pull request closed without merging**